po/fr.po

```diff
@@ -1,5 +1,723 @@
+msgstr ""
+"ATTENTION !\n"
+"Akira est encore en cours de développement et n'est pas prêt à être mis en "
+"production. Il faut s'attendre à ce que des fonctionnalités soit "
+"manquantes, qu'il y ait des bugs aléatoires et que des trous noirs "
+"s'ouvrent dans votre cuisine."
+
+#: src/Dialogs/ReleaseDialog.vala:100 src/Dialogs/SettingsDialog.vala:249
+msgid "Make a Donation"
+msgstr "Faire un don"
+
+#: src/Dialogs/ReleaseDialog.vala:109 src/Dialogs/SettingsDialog.vala:258
+msgid "Suggest Translations"
+msgstr "Suggérer des traductions"
+
+#: src/Dialogs/ReleaseDialog.vala:118 src/Dialogs/SettingsDialog.vala:267
+msgid "Report a Problem"
+msgstr "Signaler un problème"
+
+#: src/Dialogs/SettingsDialog.vala:40
+msgid "Preferences"
+msgstr "Préférences"
+
+#: src/Dialogs/SettingsDialog.vala:50 src/Dialogs/SettingsDialog.vala:73
+#: src/Dialogs/ShortcutsDialog.vala:45
+msgid "General"
+msgstr "Général"
+
+#: src/Dialogs/SettingsDialog.vala:51 src/Dialogs/SettingsDialog.vala:86
+msgid "Interface"
+msgstr "Interface"
+
+#: src/Dialogs/SettingsDialog.vala:52 src/Layouts/HeaderBar.vala:280
+msgid "Shapes"
+msgstr "Formes"
+
+#: src/Dialogs/SettingsDialog.vala:53
+msgid "About"
+msgstr "À propos"
+
+#: src/Dialogs/SettingsDialog.vala:74
+msgid "Auto Reopen Latest File:"
+msgstr "Rouvrir automatiquement le dernier fichier :"
+
+#: src/Dialogs/SettingsDialog.vala:88
+msgid "Use Dark Theme:"
+msgstr "Utiliser le thème sombre :"
+
+#: src/Dialogs/SettingsDialog.vala:92
+msgid "Invert Panels Order:"
+msgstr "Inverser l'ordre des panneaux :"
+
+#: src/Dialogs/SettingsDialog.vala:96
+msgid "Restart application to apply this change."
+msgstr "Redémarrer l'application pour appliquer ce changement."
+
+#: src/Dialogs/SettingsDialog.vala:100
+msgid "ToolBar Style"
+msgstr "Style de la barre d'outils"
+
+#: src/Dialogs/SettingsDialog.vala:102
+msgid "Show Button Labels:"
+msgstr "Afficher l'étiquette des boutons :"
+
+#: src/Dialogs/SettingsDialog.vala:106
+msgid "Use Symbolic Icons:"
+msgstr "Utiliser les icônes symboliques :"
+
+#: src/Dialogs/SettingsDialog.vala:129
+msgid "Default Colors"
+msgstr "Couleurs par défaut"
+
+#: src/Dialogs/SettingsDialog.vala:131
+msgid "Define the default style used when creating a new shape."
+msgstr ""
+"Définissez le style par défaut utilisé lors de la création d'une nouvelle "
+"forme."
+
+#: src/Dialogs/SettingsDialog.vala:136
+msgid "Fill Color:"
+msgstr "Couleur de remplissage :"
+
+#: src/Dialogs/SettingsDialog.vala:157
+msgid "Enable Border Style:"
+msgstr "Activer le style de bordure :"
+
+#: src/Dialogs/SettingsDialog.vala:160
+msgid "Border Color:"
+msgstr "Couleur de la bordure :"
+
+#: src/Dialogs/SettingsDialog.vala:181
+msgid "Border Width:"
+msgstr "Largeur de la bordure :"
+
+#: src/Dialogs/SettingsDialog.vala:209
+msgid "The Linux Design Tool"
+msgstr "L'outil de design Linux"
+
+#: src/Dialogs/SettingsDialog.vala:226
+msgid "Thanks to our awesome supporters!"
+msgstr "Merci à tous ceux qui nous soutiennent !"
```

Comment:
I think 
> Merci à toutes les merveilleuses personnes qui nous soutiennent !

would convey more the spirit of the original wording, plus it's gender neutral this way.

Reply:
Thanks for the inclusiveness, and it better conveys the idea!